packages/components/select/src/select.ts

```diff
@@ -265,25 +269,23 @@ export class Select<T = unknown> extends ObserveAttributesMixin(FormControlMixin
   override render(): TemplateResult {
     return html`
       <slot name="button"></slot>
-      <div
+      <sl-listbox
         ${anchor({
           element: this.button,
           offset: Select.offset,
           position: 'bottom-start',
-          viewportMargin: Select.viewportMargin,
-          rootMarginTop: this.rootMarginTop
+          rootMarginTop: this.rootMarginTop,
```

Comment:
In the current version of the select, when we scroll down and the select is opened, when it's not visible, the dropdown is closing and when we scroll up and the select is visible again, the dropdown is visible again as well (when the select was not closed explicitly by the user):

https://storybook.sanomalearning.design/?path=/story/form-select--hide-when-out-of-view

In the version from this PR it's not working the same, the listbox is not being shown again, when I scroll up:

https://kind-plant-088739303-1794.westeurope.1.azurestaticapps.net/?path=/story/form-select--hide-when-out-of-view

Reply:
I think we should simplify things and just hide the popover when a user starts scrolling *outside* the popover. That's what Spectrum does. Native select doesn't allow *any* scrolling when a context menu is open.
But have we discussed this before? I can't recall.

Reply:
I think we were discussing it some time ago, @Diaan right?

Reply:
https://github.com/sl-design-system/components/issues/1216

Reply:
I think i broke the behavior after using `allow-discrete` here: ```  @media (prefers-reduced-motion: no-preference) {
    transition: 0.2s cubic-bezier(0.25, 0, 0.3, 1);
    transition-behavior: allow-discrete;
    transition-property: display, opacity;
  }
```

Reply:
Because the popover is now closed when this code runs: 
```ts
export const updatePopoverVisibility = (host: HTMLElement | undefined, outOfView: boolean): void => {
  if (outOfView) {
    host?.style.setProperty('display', 'none');
  } else {
    host?.style.removeProperty('display');
  }
};
```

Reply:
Disregard this; it's the `focusout` event handler that hides the popover.